### PR TITLE
win_virtio_driver_installer: update balloon test

### PR DESCRIPTION
**Pull request by xiagao**
Define balloon test object together with execution. And in the driver
     function loop, move balloon test to the top to avoid possible conflicts.

ID: 1727

### CODE REVIEW

**Comment by xiagao** [comment]
Test pass on win10-32,win2016,win11,win2022 guest.

**Comment by xiagao** [comment]
@menli820 Could you help review this patch? thx.

**Comment by menli820** [comment]
LGTM

**Review by menli820 — APPROVED** [review]

**Comment by xiagao** [comment]
@leidwang Would you like to review this patch, thanks.

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg [hunk 1]
```diff
@@ -98,7 +98,7 @@
             numa_memdev_shm0 = mem-mem1
             numa_nodeid_shm0 = 0
             fs_binary_extra_options = " -o cache=auto"
-            test_drivers = 'viorng viostor vioscsi viofs balloon'
+            test_drivers = 'balloon viostor vioscsi viorng viofs balloon'
```
Comment (leidwang) [inline qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:101]: Hi @xiagao 

Just have a confuse, why we need 2 balloon in the test_drivers? 
Would you please help explain it more?Thanks!

Comment (xiagao) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg]: Yes, it is. 
Updated, pls help check it again, thx.

**Review by leidwang — APPROVED** [review]
LGTM, thanks @xiagao

**Comment by xiagao** [comment]
> LGTM, thanks @xiagao

Thanks for helping review.

**Comment by xiagao** [comment]
@vivianQizhu  Could you help review? Thx.

**Comment by vivianQizhu** [comment]
Hi @xiagao The code looks good to me, however for the commit msg, it seems confusing. Would you consider to update like "Define balloon test object together with execution. And in the driver function loop, move balloon test to the top to avoid possible conflicts."?

**Comment by vivianQizhu** [comment]
Also for the commit title, I would suggest "update balloon test" instead, or "balloon function test".

**Comment by xiagao** [comment]
> Also for the commit title, I would suggest "update balloon test" instead, or "balloon function test".

Sure, this is better.
I updated the code, pls have a check, thanks.

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**